src/index.html

```diff
@@ -7,8 +7,82 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      
+      <a href="https://www.moyo.ua/">
+        <img
+          src="images/logo.png"
+          alt="Moyo main logo"
+          class="logo__image"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item ">
+            <a
+              href="https://www.moyo.ua/apple"
+              class="nav__link is-active"
+            >Apple</a>
```

Comment:
```suggestion
            <a
              href="https://www.moyo.ua/apple"
              class="nav__link is-active"
            >
              Apple
            </a>
```
Formatting should look like this